constructor-manager/README.md

```diff
@@ -1,3 +1,463 @@
-# Constructor manager
+# Constructor Manager
+
+Handle (conda) constructor based bundled applications from the command line.
+
+## Requirements
+
+- conda
+- conda-lock
+- mamba
+- packaging
+- requests
+- pyyaml
+
+## Installation
+
+This package needs to be installed on the `base` conda environemnt.
+
+```bash
+conda install conda-lock mamba packaging requests pyyaml -c conda-forge -y
+pip install -e .
+```
+
+## Examples
+
+### Check for version
+
+Check for the version of the currently (latest?) installed package.
+
+```bash
+constructor-manager check-version "napari"
+```
+
+### Check for packages
+
+Check for all the packages installed in the current environment for the latest version ot the package.
+
+```bash
+constructor-manager check-packages "napari=0.4.15" --plugins-url https://api.napari-hub.org/plugins
+constructor-manager check-packages "napari=0.4.16" --plugins-url https://api.napari-hub.org/plugins
+constructor-manager check-packages "napari" --plugins-url https://api.napari-hub.org/plugins
+```
+
+### Check for updates
+
+#### For a given package
+
+```bash
+constructor-manager check-updates "napari=0.4.16=*pyside*" -c conda-forge
+constructor-manager check-updates "napari=*=*pyside*" -c conda-forge
```

Comment:
Maybe add a note (or link?) about the format of this version spec. I was not familiar with the syntax for specifying build strings and wildcards.